python/ycm/vimsupport.py

```diff
@@ -465,17 +465,55 @@ def GetIntValue( variable ):
 # start and end are objects with line_num and column_num properties
 #
 # returns the delta (in characters) to the line length applied
-#
-# PreCondition: start and end are on the same line.
 def ReplaceChunk ( start, end, replacement_text, delta ):
```

Comment:
I'll see what I can do.

However, how do we test something so closely tied to vim?